Move all relevant info to polymc-gentoo (#58)
PolyMC/polymc.github.io: src/wiki/installing/linux/gentoo.md

Gentoo ebuilds are available in the [polymc-gentoo](https://gitlab.com/flowln/polymc-gentoo) overlay, named `games-action/polymc`. If you encounter any problems with those, please open an issue on that repository first, since it is maintained by third-parties!

> There are various informations, such as install instructions and USE flag usage, [laid out on the README](https://gitlab.com/flowln/polymc-gentoo/-/blob/main/README.md). Have fun! :)

Reviewer: ```suggestion
More information, such as install instructions and USE flag usage, is [laid out in the README](https://gitlab.com/flowln/polymc-gentoo/-/blob/main/README.md). Have fun! :)
```
